### ISSUE TITLE
[field: SyncVar] {get; ServerRpc(RunLocally = true)] set;} not working correctly after updating to 2.6

### ISSUE THREAD

**Issue by Wikator** [FirstGearGames/FishNet#165]
I'm using something similar to this example code from documentation, but after updating to 2.6 (or 2.6.1) it no longer works. Removing ReadPermissions fixes it

```
[field: SyncVar(ReadPermissions = ReadPermission.ExcludeOwner)]
public string Name { get; [ServerRpc(RunLocally = true)] set; }
```

**Comment by Wikator** [comment]
Probably an issue with RunLocally, because owner is the only client that doesn't get the synchronized value

**Comment by FirstGearGames** [comment]
Need more information on what isn't working, and example code please.

**Comment by Wikator** [comment]
```
    [field: SyncVar(ReadPermissions = ReadPermission.ExcludeOwner)]
    public string Color { get; [ServerRpc(RunLocally = true)] set; }

    public string TankType { get; [ServerRpc(RunLocally = true)] set; }
```

### UI :

```
    [SerializeField]
    private List<Button> colorButtons = new();

    [SerializeField]
    private List<Button> tankTypesButtons = new();


    public override void Init()
    {
        if (Initialized)
            return;

        base.Init();

        foreach (Button button in colorButtons)
        {
            button.onClick.AddListener(() => PlayerNetworking.Instance.Color = button.name);
            Debug.Log(button.name);
        }

        foreach (Button button in tankTypesButtons)
        {
            button.onClick.AddListener(() => PlayerNetworking.Instance.TankType = button.name);
            Debug.Log(button.name);
        }
    }
```

From what I understand, RunLocally isn't working. After pressing the button, the values change on the server, and on all the remaining clients in case of the SyncVar, but not on the the client that pressed the button. On 2.5.10 it worked fine

**Comment by FirstGearGames** [comment]
2.6.0 experimental had observer bugs which were resolved in 2.6.1

**Comment by Wikator** [comment]
Yes, I also tried adding RequireOwnership = false to the RPC, didn't help

One thing I haven't mentioned is that pressing the button as a host doesn't do anything at all


Changing the variable to look like this fixes the issue

```
    [field: SyncVar]
    public string Color { get; [ServerRpc] set; }
```

**Comment by FirstGearGames** [comment]
I'll run some test and see what results I get. Thanks!

**Comment by FirstGearGames** [comment]
Resolved in 2.6.2
